src/main/java/fact/io/hdureader/ZFITSHeapReader.java

```diff
@@ -106,6 +112,29 @@ public static ZFITSHeapReader forTable(BinTable binTable) {
         return new ZFITSHeapReader(binTable);
     }
 
+    @Override
+    public void skipToRow(int num) throws IOException {
+        //calc zshrink problem
+        //num = num+1;
```

Comment:
remove or explain

Reply:
fixed the error and thus the problem with zshrink